### PR TITLE
Updated HeatMap dialogue's Plot Options to the new system.

### LINKED ISSUE #7556: Change the specific plots to the new system
I am suggesting @EstherNjeriLiberatta for this, as (I am afraid) you did the PICSA and mapping graphs very well!

a) I suggest there be a single button, just `Plot Options`.  And it has the same pull-down as for the mapping dialogue.  
b) Then the first option is also Plot Options.
c) This is followed by each of the remaining options for that dialogue, and they could be in alphabetical order.
d) I suggest they don't change for the different options at the top.  So, for example, for bar charts it is the same whether the top radio button if Frequency, Value, Treemap or Wordcloud.
e) Then simply include each of those, including label and lollipop, etc.

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
Partially fixes #7556
In this PR, I have replaced the `Tile Options` button in the `HeatMap plot` dialogue with the new `Plot Options` button which includes a drop-down.
@rdstern @africanmathsinitiative/developers  This is ready for review.

### CODE REVIEW

**Review by rdstern — DISMISSED** [review]
Looks fine, the cosmetic changes apart.  It would be good to merge this, because it will change once the choropleth maps are added.

File: instat/dlgHeatMapPlot.vb [hunk 1]
```diff
@@ -343,4 +312,46 @@ Public Class dlgHeatMapPlot
...
+                        clsNewGlobalAesFunction:=clsRaesFunction, clsNewXLabsTitleFunction:=clsXlabsFunction, clsNewScaleFillViridisFunction:=clsScaleFillViridisFunction,
+                        clsNewScaleColourViridisFunction:=clsScaleColourViridisFunction, clsNewYLabTitleFunction:=clsYlabFunction, clsNewLabsFunction:=clsLabsFunction,
+                        clsNewFacetFunction:=clsRFacetFunction, clsNewThemeFunction:=clsThemeFunction, dctNewThemeFunctions:=dctThemeFunctions, ucrNewBaseSelector:=ucrHeatMapSelector,
+                        strMainDialogGeomParameterNames:=strGeomParameterNames, clsNewCoordPolarFunction:=clsCoordPolarFunction, clsNewCoordPolarStartOperator:=clsCoordPolarStartOperator,
+                         clsNewAnnotateFunction:=clsAnnotateFunction, clsNewXScaleDateFunction:=clsXScaleDateFunction, clsNewYScaleDateFunction:=clsYScaleDateFunction, bReset:=bResetSubdialog)
+        sdgPlots.ShowDialog()
+        bResetSubdialog = False
+    End Sub
+
+    Private Sub toolStripMenuItemPlotOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemPlotOptions.Click
```
Comment (lloyddewit) [inline instat/dlgHeatMapPlot.vb:327]: Is this function identical to the function above?
If yes, then could the function above also handle the `toolStripMenuItemPlotOptions.Click` event?

File: instat/dlgHeatMapPlot.vb [hunk 2]
```diff
@@ -343,4 +312,46 @@ Public Class dlgHeatMapPlot
         End If
 
     End Sub
+
+    Private Sub cmdOptions_Click_1(sender As Object, e As EventArgs) Handles cmdOptions.Click
```
Comment (lloyddewit) [inline instat/dlgHeatMapPlot.vb:316]: what's the reason for renaming this sub to `cmdOptions_Click_1`?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@EstherNjeriLiberatta thanks, looks good, just a couple of small questions

**Comment by EstherNjeriLiberatta** [comment]
@rdstern @lloyddewit thank you for those corrections, I have made the changes you requested. This is ready for review.

**Comment by lloyddewit** [comment]
@rdstern if you can retest/approve, then we can merge, thanks

**Review by rdstern — APPROVED** [review]
@EstherNjeriLiberatta don't mind this, but it presumably will also change with your choropleth pull request?

**Comment by EstherNjeriLiberatta** [comment]
@lloyddewit

**Pull request closed without merging**